### DISCUSSION TITLE
UI-Design Consistency: Harmonizing Iconography

### DISCUSSION THREAD

**Discussion by alexanderbuergin** [craftcms/cms#14180] [Ideas] [answered]
I've noticed that with the latest Craft CMS update, there have been new icons introduced within the settings section. While the fresh icons add a distinct visual touch, they seem to stand out from the established icon language used elsewhere in the system. This disparity in design can potentially affect the user's visual experience and navigational flow.

The current icon set exhibits a variety of styles that, although individually appealing, lack cohesion when viewed as part of the entire CMS interface. A unified icon language is essential not only for aesthetic harmony but also for creating an intuitive user interface that users can navigate with ease and confidence.

To address this, I propose a review and redesign of the icon set to ensure stylistic consistency across all interfaces of the CMS. 

This could involve:
- Evaluating current icons for visual consistency in terms of line weight, icon geometry, and fill styles.
- Redesigning outliers to align with the most prevalent icon style within the CMS.
- Establishing a design guideline for current and future icons to preserve uniformity.

---

**Line-based icons**

<img width="840" alt="Bildschirmfoto 2024-01-23 um 21 35 00" src="https://github.com/craftcms/cms/assets/85960512/31bf7438-e620-4e8f-a872-8253a0dd7282">

**Filling based icons**

<img width="578" alt="Bildschirmfoto 2024-01-23 um 21 35 09" src="https://github.com/craftcms/cms/assets/85960512/26a75bb5-8bee-458f-aec8-547b3e8ea39a">

**Comment by brandonkelly** [comment, marked as answer]
They’re two different weight variants of the same icon set (Font Awesome).

I chose the lighter icons for the Settings page because the normal `solid` variant looks pretty jarring when blown up to that size:

![The system setting icons using the solid variant](https://github.com/craftcms/cms/assets/47792/28092b61-64a4-44c2-b3e2-55d4d071837b)
